src/scheduler/kubernetes/scheduler.py

```diff
@@ -335,6 +335,16 @@ def upload_console(self, job_id):
         """, [job_id])
         cursor.close()
 
+    def handle_inactive_cluster_queued_jobs(self):
+        cursor = self.conn.cursor()
+        cursor.execute("""
```

Comment:
I think this may break some jobs. When we have jobs with dependencies and the parent job produces some output we forward the data to the cluster on which the child job is executed. This can be done, because we know at data upload time the cluster on which the child is supposed to run. But with this change we change the cluster after we have uploaded the data. So the child job may end up on a cluster which does not have the input data and therefore fail.